README.md

```diff
@@ -41,6 +41,7 @@ You can either use the provided `Ansible` playbook or run the steps manually
           ansible_user: <Username to connect with>
           ansible_ssh_private_key_file: <Location of your Private SSH key>
           ansible_become: <Set to false if connecting as "root", otherwise remove>
+          ansible_python_interpreter: /usr/libexec/platform-python
```

Comment:
Why did you need to set this?

Reply:
This is done in order to use the same python version, even if the python command in PATH changed to another version (modified by dev-scripts)